yoga/Yoga.c

```diff
@@ -229,8 +229,6 @@ static YGLogger gLogger = &YGDefaultLog;
 static inline const YGValue *YGComputedEdgeValue(const YGValue edges[YGEdgeCount],
                                                  const YGEdge edge,
                                                  const YGValue *const defaultValue) {
-  YG_ASSERT(edge <= YGEdgeEnd, "Cannot get computed value of multi-edge shorthands");
```

Comment:
It is not needed i guess. but getting the computed value of "all" or "horizontal edge just does not make sense. e.g.
```
node.marginHorizontal = 10;
node.marginLeft = 20;
node.marginRight = 30;
```
At this point resolving `marginHorizontal` does not make much sense. I don't think this should be fixed by removing the assert. If we want to return the value set then we should instead have another function to do so.